### ISSUE TITLE
List item overrides header item

### ISSUE THREAD

**Issue by levouh** [akinsho/org-bullets.nvim#10]
The new addition of symbols for list items overrides "level 1" headers (in the image below the level 1 headline bullet should be much larger):

![image](https://user-images.githubusercontent.com/31262046/143812437-03ac5145-2ca8-44a1-9075-67b7f8bc95b8.png)

Creating an issue to see whether or not this was intentional first, but would it be ideal to have configuration to setup what list items users want to support? For example, I only ever use `-` for lists. Happy to look into opening a _PR_ for this.

**Comment by levouh** [comment]
No updates as far as I can tell, but this has since seemingly fixed itself. Will re-open if need be.

**Comment by levouh** [comment]
Re-opening this with an easy way to reproduce it.

1. Create an _Org_ file, say:
```
cat "* Test item" > /tmp/test.org
```
2. Setup the following configuration, say in `/tmp/init.lua`:
```lua
vim.cmd([[
  set rtp=$VIMRUNTIME
  packadd nvim-treesitter
  packadd orgmode
  packadd org-bullets.nvim
]])

local parser_config = require("nvim-treesitter.parsers").get_parser_configs()

parser_config.org = {
  install_info = {
    url = "https://github.com/milisims/tree-sitter-org",
    revision = "main",
    files = { "src/parser.c", "src/scanner.cc" },
  },
  filetype = "org",
}

require("nvim-treesitter.configs").setup({
  highlight = {
    enable = true,
    disable = { "org" },
    additional_vim_regex_highlighting = { "org" },
  },
  ensure_installed = { "org" },
})

require("orgmode").setup({
  org_agenda_files = { "~/org/**/*" },
  org_default_notes_file = "~/org/refile.org",
})
```
3. Open it as follows:
```
nvim -u /tmp/init.lua --noplugin /tmp/test.org
```

Observe that sometimes you get:

![image](https://user-images.githubusercontent.com/31262046/145153919-6cab0cfc-1425-4cc2-9384-23f348cf1867.png)

and other times you get:

![image](https://user-images.githubusercontent.com/31262046/145153986-65529fbd-f1d8-4ea5-b454-e95a10da0bcd.png)

The latter would be what I'd expect to see every time.

**Comment by levouh** [comment]
I imagine this is just a byproduct of conflicting regex that really can't be avoided, say [here](https://github.com/akinsho/org-bullets.nvim/blob/73407e765c65006bf1f7740e8d4fb4450a82aa0b/lua/org-bullets.lua#L81) and [here](https://github.com/akinsho/org-bullets.nvim/blob/73407e765c65006bf1f7740e8d4fb4450a82aa0b/lua/org-bullets.lua#L96).

@akinsho would you accept a simple _PR_ to override the symbol for different kinds of list items? One solution for me (since I never use `*` style list items) would be to just make the symbol for list items with `*` match headline items with `*`. Any other better solutions you can think of?

**Comment by akinsho** [comment]
@levouh thanks for raising this. Yeah, the recent PR probably caused this to be overridden. I guess a fix could be only applying the second styling if the `*` is not top level, I'm not sure if that's a good fix or if people would create top level list items. I don't think a solution that just forces them to match headlines is a good one, since that will likely cause issues for other people.

**Comment by levouh** [comment]
>I don't think a solution that just forces them to match headlines is a good one, since that will likely cause issues for other people

I agree, take a look at my fork (I can also just make a PR for it) where I've added an option to allow users to override the characters that make up "bullets". For me, I only every use `-`, so I can just set the regex to only match to `-` characters and it works for _me_, but other users can set it as they please.

**Comment by akinsho** [comment]
This should now be resolved since I use treesitter to determine highlights now so anything that is classed a bullet in the TS grammar will have a different highlight to the stars
